partcad/src/partcad/project_factory_local.py

```
@@ -29,7 +27,7 @@ def __init__(self, ctx, parent, config):
 
         self.path = self.import_config_path
 
-        if not self.maybe_empty:
+        if not self.can_be_empty:
             if not os.path.exists(self.import_config_path):
```

Ruff (0.8.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)